src/main/java/com/zaxxer/hikari/metrics/micrometer/MicrometerMetricsTracker.java

```diff
@@ -40,51 +39,49 @@
 
    MicrometerMetricsTracker(final String poolName, final PoolStats poolStats, final MeterRegistry meterRegistry)
    {
-      this.connectionObtainTimer = meterRegistry
-         .timerBuilder(METRIC_NAME_WAIT)
+      this.connectionObtainTimer = Timer.builder(METRIC_NAME_WAIT)
+         .description("Connection acquired time")
```

Comment:
"Connection acquired time" → "Connection acquire time"

Reply:
I changed to `acquire`, however I copied from the Prometheus one, so that one still reads a `acquired`